### DISCUSSION TITLE
intelligent building IoT

### DISCUSSION THREAD

**Discussion by YXGuan** [microsoft/IoT-For-Beginners#276] [Ideas]
Hello there,

I would like to start a curriculum about **building automation system** ( BAS ) and how to implement IoT devices for more energy efficient buildings, but don't know where to get started. Any leads would be much appreciated.

Looking forward to hearing from you.

**Comment by jlooper** [comment]
hi, curriculum building takes planning and a team, we believe. I recommend gathering a group and creating a project on GitHub (use the automated GitHub projects) to plan your lessons.

Before the actual lesson planning, however, you need to set your pedagogy. There are a lot of discussions on how to teach on cultofpedagogy.com - let me know if you'd like a bibliography on curriculum design
